### PR TITLE
Fix “metadata for X field is in Table Y” sentences

### PR DESCRIPTION
**Pull request by Jayman2000**
It also adds one of those sentences to 10.1.1 since 10.1.1 was missing
one.
Signed-off-by: Jason Yundt <swagfortress@gmail.com>

### CODE REVIEW

File: chapters/package-information.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ EXAMPLE 2 RDF: Property `spdx:name` in class `spdx:Package`
 
 ### 7.2.1 Description
 
-Uniquely identify any element in an SPDX document which may be referenced by other elements. These may be referenced internally and externally with the addition of the SPDX document identifier. The metadata for the SPDX identifier field is shown in Table 14.
+Uniquely identify any element in an SPDX document which may be referenced by other elements. These may be referenced internally and externally with the addition of the SPDX document identifier. The metadata for the package SPDX identifier field is shown in Table 14.
 
 **Table 14 — Metadata for the SPDX identifier field**
```
Comment (seabass-labrax) [inline chapters/package-information.md:43]:
```suggestion
**Table 14 — Metadata for the package SPDX identifier field**
```

Maybe also change this table heading for consistency?

File: chapters/package-information.md [hunk 2]
```diff
@@ -925,7 +925,7 @@ The options to populate this field are limited to:
 
   - the SPDX document creator has intentionally provided no information (no meaning should be implied by doing so).
 
-The metadata for all license information from files field is shown in Table 26.
+The metadata for the all license information from files field is shown in Table 26.
```
Comment (seabass-labrax) [inline chapters/package-information.md:928]: Sorry; I don't understand what the purpose of this line's change is.

**Review by seabass-labrax — CHANGES_REQUESTED** [review]

File: chapters/package-information.md (hunk 1 above)
Comment (Jayman2000) [reply chapters/package-information.md:43]: Good idea. I’ll include that change in the next force push.

**Review by kestewart — APPROVED** [review]

**Comment by kestewart** [comment]
Thanks for the fixes!

**Pull request merged**